apple/model.go

```diff
@@ -88,3 +114,12 @@ type RefreshResponse struct {
 	// A more detailed precision about the current error.
 	ErrorDescription string `json:"error_description"`
 }
+
+// RevokeResponse is based of https://developer.apple.com/documentation/sign_in_with_apple/revoke_tokens
+type RevokeResponse struct {
+	// Used to capture any error returned by the endpoint. Do not trust the response if this error is not nil
```

Comment:
```suggestion
	// Used to capture any error returned by the endpoint
```

Since there is no other payload other than errors, I think we can remove this part

Reply:
I'll do it from my end since its a very simple comment change